sysutils/cloudmonkey/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/apache/cloudstack-cloudmonkey 6.0.0
+name                cloudmonkey
+
+categories          sysutils
+license             Apache-2
+
+description         CloudMonkey is a command line interface for Apache \
+                    Cloudstack.
+
+long_description    {*}${description} CloudMonkey can be used both as an \
+                    interactive shell, and as a command line tool which \
+                    simplifies Apache CloudStack configuration and management.
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+# The build process wants to see a git checkout.
+fetch.type          git
+
+build.cmd           make
+build.target        all
+
+installs_libs       no
+
+destroot {
+    xinstall -m 755 ${worksrcpath}/bin/cmk ${destroot}${prefix}/bin/
+}
+
+notes               "${name} is installed as cmk"
```

Comment:
Does it need to show these notes? Usually a port should not print a note telling the user what was installed. A user who does not know what was installed is expected to run `port contents` to find out.

Reply:
I find the notes very helpful, especially for new users who wouldn't be aware of `port contents`. I can remove if that's the harder preference.

Reply:
Hm. Remember that notes are printed after every upgrade. When you upgrade 10 outdated ports, do you really want to see 10 notes telling you what was installed? You were already using the ports; you probably already know. Or the ports were installed as dependencies of something else so you don't care. Won't that make you more likely to tune out the notes entirely and potentially miss other relevant information?

I could understand somewhat if the port were renaming the installed files in a way that users would not expect. But in this case cloudmonkey's own build system calls the program cmk, so that's what a user should expect it to be.

Reply:
Alright, notes have been removed.